### PR TITLE
Create tracing.md

### PR DESCRIPTION
**Pull request by erikperttu**
This is a first basic draft for adding tracing. This is for using OpenCensus and aws x-ray so maybe it should be added to another section/file. There are no screenshots in this one, I could use the ones from https://github.com/johanbrandhorst/grpc-gateway-boilerplate/tree/add-tracing however that is from another repo. It would not be to hard to add an actual example of this in /examples/internal. Please let me know what to change/remove.

### CODE REVIEW

**Comment by erikperttu** [comment]
@googlebot I signed it!

**Comment by johanbrandhorst** [comment]
There's a section on using `OpenTracing` in [`customizingyourgateway`](https://github.com/grpc-ecosystem/grpc-gateway/blob/v2/docs/_docs/customizingyourgateway.md#opentracing-support), could you move that across to the new file as well?

**Comment by erikperttu** [comment]
> There's a section on using `OpenTracing` in [`customizingyourgateway`](https://github.com/grpc-ecosystem/grpc-gateway/blob/v2/docs/_docs/customizingyourgateway.md#opentracing-support), could you move that across to the new file as well?

Move and remove from this one or leave it here and also add to the customization file?

**Comment by johanbrandhorst** [comment]
> > There's a section on using `OpenTracing` in [`customizingyourgateway`](https://github.com/grpc-ecosystem/grpc-gateway/blob/v2/docs/_docs/customizingyourgateway.md#opentracing-support), could you move that across to the new file as well?
> 
> Move and remove from this one or leave it here and also add to the customization file?

Move the existing section in customizing your gateway to this new file.

File: docs/_docs/tracing.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        }
+        return startOptions
+    },
+}
+```
+
+4. No global configuration means we have to use the [per span sampler](https://opencensus.io/tracing/sampling/#per-span-sampler)
+
+
+##### A method we __want__ to trace
```
Comment (johanbrandhorst) [inline docs/_docs/tracing.md:101]: Will this be correctly rendered on the docs site? I'm not sure. Maybe best to remove the italics?

Comment (erikperttu) [reply docs/_docs/tracing.md]: removed it, it does not look like github does that type of bold text

**Comment by johanbrandhorst** [comment]
Thanks for your contribution! Could you cherry pick this against v2 please?

**Comment by achew22** [comment]
I believe this was merged again v2 already, could you do it against master?

**Pull request merged**